<ui version="4.0" >
 <class>modCalcAltAz</class>
 <widget class="QWidget" name="modCalcAltAz" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Horizontal Coordinates</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="KTabWidget" name="tabWidget3" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="interact" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>498</width>
        <height>488</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="1" >
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QDateTimeEdit" name="DateTime" />
           </item>
           <item>
            <widget class="KPushButton" name="NowButton" >
             <property name="text" >
              <string>Now</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="LocationLabel" >
           <property name="text" >
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="KPushButton" name="LocationButton" >
           <property name="text" >
            <string>Greenwich, United Kingdom</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="DateLabel" >
           <property name="text" >
            <string>Date and time:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>411</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Equatorial Coordinates (J2000)</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="2" column="0" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1" >
             <widget class="KPushButton" name="ObjectButton" >
              <property name="text" >
               <string>Select Object...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="dmsBox" name="Dec" >
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="focusPolicy" >
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="dmsBox" name="RA" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="focusPolicy" >
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="decLabel" >
              <property name="text" >
               <string>Declination:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="raLabel" >
              <property name="text" >
               <string>Right ascension:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>Horizontal Coordinates</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>8</number>
            </property>
            <item>
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="1" column="1" >
               <widget class="dmsBox" name="Alt" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="focusPolicy" >
                 <enum>Qt::StrongFocus</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="dmsBox" name="Az" >
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="focusPolicy" >
                 <enum>Qt::StrongFocus</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0" >
               <widget class="QLabel" name="azLabel" >
                <property name="text" >
                 <string>Azimuth:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="elLabel" >
                <property name="text" >
                 <string>Altitude:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>840</width>
        <height>420</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>8</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Select Fields in Input File</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>8</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="3" column="0" >
               <widget class="QCheckBox" name="decCheckBatch" >
                <property name="text" >
                 <string>Declination:</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1" >
               <widget class="dmsBox" name="decBoxBatch" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="focusPolicy" >
                 <enum>Qt::StrongFocus</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QCheckBox" name="raCheckBatch" >
                <property name="text" >
                 <string>Right ascension:</string>
                </property>
                <property name="checked" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QCheckBox" name="dateCheckBatch" >
                <property name="text" >
                 <string>Date:</string>
                </property>
                <property name="checked" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0" >
               <widget class="QCheckBox" name="epochCheckBatch" >
                <property name="text" >
                 <string>Epoch:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" >
               <widget class="dmsBox" name="raBoxBatch" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="focusPolicy" >
                 <enum>Qt::StrongFocus</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0" >
               <widget class="QCheckBox" name="utCheckBatch" >
                <property name="text" >
                 <string>UT:</string>
                </property>
                <property name="checked" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QTimeEdit" name="utBoxBatch" />
              </item>
              <item row="4" column="1" >
               <widget class="KLineEdit" name="epochBoxBatch" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QDateEdit" name="dateBoxBatch" />
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <layout class="QGridLayout" >
                <property name="margin" >
                 <number>0</number>
                </property>
                <property name="spacing" >
                 <number>6</number>
                </property>
                <item row="3" column="1" >
                 <widget class="dmsBox" name="elBoxBatch" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize" >
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="focusPolicy" >
                   <enum>Qt::StrongFocus</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="dmsBox" name="longBoxBatch" >
                  <property name="enabled" >
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize" >
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="focusPolicy" >
                   <enum>Qt::StrongFocus</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" >
                 <widget class="QCheckBox" name="elCheckBatch" >
                  <property name="text" >
                   <string>Altitude:</string>
                  </property>
                  <property name="checked" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" >
                 <widget class="QCheckBox" name="azCheckBatch" >
                  <property name="text" >
                   <string>Azimuth:</string>
                  </property>
                  <property name="checked" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" >
                 <widget class="QCheckBox" name="latCheckBatch" >
                  <property name="text" >
                   <string>Latitude:</string>
                  </property>
                  <property name="checked" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" >
                 <widget class="QCheckBox" name="longCheckBatch" >
                  <property name="text" >
                   <string>Longitude:</string>
                  </property>
                  <property name="checked" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="dmsBox" name="azBoxBatch" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize" >
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="focusPolicy" >
                   <enum>Qt::StrongFocus</enum>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" >
                 <widget class="dmsBox" name="latBoxBatch" >
                  <property name="enabled" >
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize" >
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="focusPolicy" >
                   <enum>Qt::StrongFocus</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>16</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Fields in Output File Plus Result</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>8</number>
          </property>
          <item>
           <widget class="QRadioButton" name="allRadioBatch" >
            <property name="text" >
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inputRadioBatch" >
            <property name="text" >
             <string>Only parameters in input file</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Files</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="KLineEdit" name="OutputLineEditBatch" />
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="OutputButtonBatch" >
            <property name="toolTip" >
             <string>File for the output data</string>
            </property>
            <property name="text" >
             <string>Output File...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="KLineEdit" name="InputLineEditBatch" />
          </item>
          <item row="0" column="0" >
           <widget class="QPushButton" name="InputButtonBatch" >
            <property name="toolTip" >
             <string>File with the input data</string>
            </property>
            <property name="text" >
             <string>Input File...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>151</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="runButtonBatch" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget3</tabstop>
  <tabstop>NowButton</tabstop>
  <tabstop>LocationButton</tabstop>
  <tabstop>RA</tabstop>
  <tabstop>Dec</tabstop>
  <tabstop>ObjectButton</tabstop>
  <tabstop>Az</tabstop>
  <tabstop>Alt</tabstop>
  <tabstop>decCheckBatch</tabstop>
  <tabstop>decBoxBatch</tabstop>
  <tabstop>raCheckBatch</tabstop>
  <tabstop>dateCheckBatch</tabstop>
  <tabstop>epochCheckBatch</tabstop>
  <tabstop>raBoxBatch</tabstop>
  <tabstop>utCheckBatch</tabstop>
  <tabstop>utBoxBatch</tabstop>
  <tabstop>epochBoxBatch</tabstop>
  <tabstop>elBoxBatch</tabstop>
  <tabstop>longBoxBatch</tabstop>
  <tabstop>elCheckBatch</tabstop>
  <tabstop>azCheckBatch</tabstop>
  <tabstop>latCheckBatch</tabstop>
  <tabstop>longCheckBatch</tabstop>
  <tabstop>azBoxBatch</tabstop>
  <tabstop>latBoxBatch</tabstop>
  <tabstop>allRadioBatch</tabstop>
  <tabstop>inputRadioBatch</tabstop>
  <tabstop>OutputLineEditBatch</tabstop>
  <tabstop>OutputButtonBatch</tabstop>
  <tabstop>InputLineEditBatch</tabstop>
  <tabstop>InputButtonBatch</tabstop>
  <tabstop>runButtonBatch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
